hive.go

```diff
@@ -228,17 +230,17 @@ func (r *simRunner) runSimulatorAPIDevMode(ctx context.Context, endpoint string)
 		return err
 	}
 
-	listener, err := net.ListenTCP("tcp4", addr)
-	if err != nil {
-		log15.Error(fmt.Sprintf("failed to start TCP server on %s", addr), "err", err)
-		return err
-	}
+	// listener, err := net.ListenTCP("tcp4", addr)
+	// if err != nil {
+	// 	log15.Error(fmt.Sprintf("failed to start TCP server on %s", addr), "err", err)
+	// 	return err
+	// }
 
 	log15.Info(fmt.Sprintf("simulator API listening at %s", addr))
-	server := &http.Server{Handler: tm.API()}
-	defer shutdownServer(server)
-
-	go server.Serve(listener)
+	// server := &http.Server{Handler: tm.API()}
```

Comment:
Shouldn't you remove that log statement too? It's not true any longer (right?)

Reply:
I need to fix this, just commented these lines to make it compile :)